doc/code/qml_qaoa.rst

```diff
@@ -100,3 +100,4 @@ Cycle Optimization
     :no-heading:
     :no-inheritance-diagram:
     :no-inherited-members:
+    :skip: kron
```

Comment:
Yes double checked by removing and rebulding docs, `kron` appears under Functions in the docs. No issues from `csc_matrix`.

Reply:
Have been able to remove since `kron` was only used in `martix()` which was moved to `test_qaoa.py` in this commit:
47a4657a424b4d44c29d3d6ab7fbf1e5b42d581a

Reply:
Nice!